CONTRIBUTING.rst

```diff
@@ -3,8 +3,6 @@ Contributing to MLflow
 We welcome community contributions to MLflow. This page describes how to develop/test your changes
 to MLflow locally.
 
-Python
```

Comment:
Removed this because there's a separate section also titled "Python" (with analogous "Java" and "R" sections)  discussing the dev workflow for the Python package further below